pkg/virt-controller/watch/pool.go

```diff
@@ -408,7 +441,7 @@ func (c *PoolController) Run(threadiness int, stopCh <-chan struct{}) {
 	log.Log.Info("Starting pool controller.")
 
 	// Wait for cache sync before we start the pool controller
-	cache.WaitForCacheSync(stopCh, c.poolInformer.HasSynced, c.vmInformer.HasSynced)
+	cache.WaitForCacheSync(stopCh, c.poolInformer.HasSynced, c.vmInformer.HasSynced, c.vmiInformer.HasSynced, c.revisionInformer.HasSynced)
```

Comment:
You're waiting for the vmiInformer twice here, just a copy-paste error probably.

Reply:
oops, guess I missed an 'i'.